src/components/History/Widget/WidgetCardTransaction.tsx

```diff
@@ -105,10 +105,14 @@ const WidgetCardTransaction = ({
       if (!details?.transaction || !hash || iTokenDetailMissing) {
         return <WidgetCardLoading />
       }
+      const recoverAmount = (
+        Number(details!.recoverDetails?.amount ?? "0") /
```

Comment:
Maybe big int to avoid possible issues with js numbers?

Reply:
This going to be separate commit with update for entire app.